components/graph-line.vue

```diff
@@ -21,6 +21,7 @@
       <v-chart
         :options="graphData()"
         :autoresize="true"
+        height="100%"
```

Comment:
in css?

Reply:
it's an attribute for the vue echarts comp